pkgs/tools/networking/isync/default.nix

```diff
@@ -1,32 +1,69 @@
-{ lib, stdenv, fetchurl, pkg-config, perl
-, openssl, db, cyrus_sasl, zlib
-, Security
-# Disabled by default as XOAUTH2 is an "OBSOLETE" SASL mechanism and this relies
-# on a package that isn't really maintained anymore:
-, withCyrusSaslXoauth2 ? false, cyrus-sasl-xoauth2, makeWrapper
+{
+  lib,
+  stdenv,
+  fetchgit,
+  pkg-config,
+  perl,
+  openssl,
+  db,
+  cyrus_sasl,
+  zlib,
+  perl538Packages,
+  autoreconfHook,
+  Security,
+  # Disabled by default as XOAUTH2 is an "OBSOLETE" SASL mechanism and this relies
+  # on a package that isn't really maintained anymore:
+  withCyrusSaslXoauth2 ? false,
+  cyrus-sasl-xoauth2,
+  makeWrapper,
 }:
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "isync";
   version = "1.5.0";
```

Comment:
This should be `1.5.0-unstable-2024-09-29`  (see https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning).